nf_core/__main__.py

```diff
@@ -502,10 +502,10 @@ def create_module(ctx, tool, dir, author, label, meta, no_meta, force, conda_nam
     """
     Create a new DSL2 module from the nf-core template.
 
-    If <directory> is a pipeline, this function creates a file called
+    If the working directory is a pipeline, this function creates a file called
```

Comment:
maybe `specified` rather than working? The `work` directory is a specific thing in nextflow so might be (temporarily) confusing? Unless you agreed on the terminology elsewhere

Reply:
Yes you are right, "specified" is better than "working". I've updated it.